content/en/technical-overview/architecture/cht-core.md

```diff
@@ -1,36 +1,73 @@
 ---
-title: "Architecture of CHT Instances"
-linkTitle: "Architecture"
-weight: 1
+title: "CHT Core"
+linkTitle: "CHT Core"
+weight: 2
 description: >
-  Overview of CHT components and interactions
+  Technical overview of the CHT Core Framework
 relatedContent: >  
-  technical-overview/cht-watchdog
-  technical-overview/cht-sync
+  technical-overview/architecture/cht-watchdog
+  technical-overview/architecture/cht-sync
 aliases:
    - /core/overview/architecture
 ---
 
-## Overview
-
-<!-- make updates to this diagram on the google slides:            -->
-<!-- https://docs.google.com/presentation/d/1j4jPsi-gHbiaLBfgYOyru1g_YV98PkBrx2zs7bwhoEQ/ -->
-
-{{< figure src="architecture.png" link="architecture.png" caption="Data Flows" >}}
+The CHT Core Framework is the primary component of the CHT. The server comes with authentication, role based authorization, data security, and a range of protected data access endpoints. Read more detail in [cht-core GitHub repository](https://github.com/medic/cht-core).
+
+```mermaid
+graph TB
+  classDef container stroke:#63a2c6, fill:#eef5f9, color:#000
+  
+  subgraph CHT_Core_Framework["<div style="width: 23em; height:2em; display:flex; justify-content: flex-start; align-items:flex-end;">Server</div>"]
+    direction TB
+    NGINX(NGINX):::container
+    API(API):::container
+    Sentinel(Sentinel):::container
+    HAProxy(Haproxy):::container
+    HealthCheck(HealthCheck):::container
+    CouchDB[(CouchDB)]:::container
+    Nouveau[(Nouveau)]:::container
+  end
+
+  Upgrade["Upgrade Service"]:::container
+
+  subgraph Client["<div style="width: 28em; height:2em; display:flex; justify-content: flex-start; align-items:flex-end;">Client (Android & Browser)</div>"]
+    subgraph Webapp["Webapp"]
+      PouchDB[(PouchDB)]:::container
+    end
+
+    Admin["App Management"]:::container
+end
+  
+  Webapp --> NGINX
+  Admin --> NGINX
+  NGINX --> API
+  API <--> Upgrade
+  Sentinel --> HAProxy
+  API --> HAProxy
+  HAProxy --> CouchDB
+  HealthCheck --> CouchDB
+  HAProxy --> HealthCheck
+  CouchDB --> Nouveau
+
+  style CHT_Core_Framework fill: transparent
+  style Client fill: transparent
+  style Webapp fill: transparent
+```
+
+{
```

Comment:
These lines appear out of context. Are they meant to be here?
<img width="709" alt="Screenshot 2025-05-27 at 10 34 59" src="https://github.com/user-attachments/assets/6e7fcf3c-17af-4e46-b862-17e7769c99f0" />

Reply:
oops removed